### ISSUE TITLE
Pioneer BDR-XS06: Drive spins up but times out

### ISSUE THREAD

**Issue by voltagex** [saramibreak/DiscImageCreator#12]
```
PS C:\temp> DiscImageCreator.exe dvd g stranglehold1.iso 8
AppVersion
        x86, AnsiBuild, Mar  4 2018 15:29:36
CurrentDirectory
        C:\temp
WorkingPath
         Argument: stranglehold1.iso
         FullPath: C:\temp\stranglehold1
            Drive: C:
        Directory: \temp\
         Filename: stranglehold1
        Extension: .iso
Start time: 2018-03-08(Thu) 21:17:15
Set the drive speed: 11075KB/sec
[F:ReadDVDForFileSystem][L:24] GetLastError: 121, The semaphore timeout period has expired.

Please wait for 40000 milliseconds until the device is returned
End time: 2018-03-08(Thu) 21:18:55
```
Even at 1x it doesn't seem to work.

PS: I'd love to disable the tune that's played at exit...

**Comment by saramibreak** [comment]
Does it occur only with this disc? Or all disc?

**Comment by voltagex** [comment]
Seems like all discs - although I don't have too many DVDs to test

**Comment by saramibreak** [comment]
Please upload the log (.txt file). And if you have other drive, tell me the result.

**Comment by voltagex** [comment]
How do I create a log?

**Comment by saramibreak** [comment]
Is there `stranglehold1_drive.txt` `stranglehold1_disc.txt`in "C:\temp\"?

**Comment by voltagex** [comment]
[stranglehold1_disc.txt](https://github.com/saramibreak/DiscImageCreator/files/1797773/stranglehold1_disc.txt)
[stranglehold.txt](https://github.com/saramibreak/DiscImageCreator/files/1797774/stranglehold.txt)
[stranglehold1_cmd.txt](https://github.com/saramibreak/DiscImageCreator/files/1797775/stranglehold1_cmd.txt)

As requested

**Comment by saramibreak** [comment]
Uploaded test  version.
http://www.mediafire.com/file/eq80y20l9cwf48f/DiscImageCreator_test.7z
If same error occurs, I don't know the reason.

**Comment by voltagex** [comment]
Same issue unfortunately

**Comment by saramibreak** [comment]
This problem is perhaps fixed at 20180619 version.

**Comment by voltagex** [comment]
Aha. Coming back to this almost a year later: 
First issue: caused by using a USB connected drive, I think

Solved by using the 20190326 version - even setting the drive speed works now.
